app/views/players/edit.html.slim

```diff
@@ -2,16 +2,7 @@ h1.title
   | EDIT PLAYER
 .form
   = form_for :player, url: player_path(@player), method: :patch do |f|
```

Comment:
Actually you can just put this line (and the line above with `.form`) into the form partial, and just pass the `@player` variable into that partial. So you don't have to pass a form object around. 

That will allow you to reuse the entire form on other pages if you need to next time.